packages/components/Notification/src/NotificationTokens.ios.ts

```diff
@@ -104,8 +104,8 @@ export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme>
     fontSize: 15,
     fontWeight: '400',
     minHeight: 52,
-    padding: 16,
-    paddingVertical: 12,
+    padding: t.spacing.m,
```

Comment:
i must have missed the train on this one. but ".m" or "s" is it too abbreviate it? is it meant to be medium or small?

Reply:
This abbreviation is by design.